### PR TITLE
Zak/add linter

### PR DESCRIPTION
**Pull request by ZakMiller**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "extends": "standard",
+  "plugins": [
+        "standard",
+        "promise"
```
Comment (SylvanSign) [inline .eslintrc.js]: can you beautify this?

**Review by SylvanSign — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -24,7 +24,12 @@
     "babel-preset-es2015": "^6.22.0",
     "babelify": "^7.3.0",
     "browserify": "^14.0.0",
+    "eslint": "^3.15.0",
+    "eslint-config-standard": "^6.2.1",
+    "eslint-plugin-promise": "^3.4.0",
+    "eslint-plugin-standard": "^2.0.1",
     "gulp": "^3.9.1",
+    "gulp-eslint": "^3.0.1",
     "gulp-jsbeautifier": "^2.0.4",
```
Comment (SylvanSign) [inline package.json]: remove this pls

**Comment by ZakMiller** [comment]
Should be A-OK now

File: package.json (hunk 2 above)
Comment (SylvanSign) [reply package.json]: and the gulp beautify task

**Review by SylvanSign — CHANGES_REQUESTED** [review]

File: gulpfile.js [hunk 3]
```diff
@@ -1,42 +1,31 @@
...
-
-// Task implementations
-function beautify() {
-  gulp.src(globsToBeautify, {
-      base: './'
-    })
-    .pipe(prettify({
-      eol: os.EOL, // \r\n on Windows, \n on POSIX
+function lint() {
+  return gulp.src(['gulpfile.js', 'server.js', 'public/scripts/main.js'])
```
Comment (SylvanSign) [inline gulpfile.js]: better to have 
```js
return gulp.src(['*.js', 'public/scripts/*.js'])

File: gulpfile.js [hunk 4]
```diff
@@ -1,42 +1,31 @@
...
-
-// Task implementations
-function beautify() {
-  gulp.src(globsToBeautify, {
-      base: './'
-    })
-    .pipe(prettify({
-      eol: os.EOL, // \r\n on Windows, \n on POSIX
+function lint() {
+  return gulp.src(['*.js', 'public/scripts/*.js'])
```
Comment (SylvanSign) [inline gulpfile.js:22]: do you need to return this or not? I had an issue where returning from `build` caused me some issues, but I don't really understand gulp that well

**Review by SylvanSign — APPROVED** [review]

**Pull request merged**